Add crowdin badge and information in readme (#3990)
Ryujinx/Ryujinx: README.md

For our Local Wireless and LAN builds, see our [Multiplayer: Local Play/Local Wireless Guide
](https://github.com/Ryujinx/Ryujinx/wiki/Multiplayer-(LDN-Local-Wireless)-Guide).

> Avalonia UI is available in various localized translations. See [crowdin.com/ryujinx](https://crwd.in/ryujinx) for more informations about translations.

Reviewer: ```suggestion
Avalonia UI comes with translations for various languages. See [Crowdin](https://crwd.in/ryujinx) for more information about them.
```
How does that sound?

Reviewer 2: I'd subtract "about them" but otherwise I like this.
